Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,46 @@
-function setAlarm() {}
+let timeRemaining;
+let timer;
+
+document.getElementById("set").addEventListener("click", () => {
+  setAlarm();
+});
+
+document.getElementById("stop").addEventListener("click", () => {
+  clearInterval(timer);
+  document.getElementById("set").innerText = "Set Alarm";
+  document.getElementById("timeRemaining").innerText = "Time Remaining: 00:00";
```

Comment:
If we wanted to change the text from "Time Remaining" to "Time Left", we'd currently need to change this in three places. Can you work out how to make it so we'd only need to change it in one place?